### PR TITLE
add puredata to caskroom

### PR DESCRIPTION
**Pull request by megrimm**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There’s trailing whitespace that needs to be removed.

File: Casks/pd.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  sha256 'fad5a77f01f25fb6b8c73e67d5d7ab3cc3ca87b5eb46d2fbe0be0e30d3f64823'
+
+  url "http://msp.ucsd.edu/Software/pd-#{version}-64bit.mac.tar.gz"
+  name 'Pd'
+  homepage 'http://msp.ucsd.edu/software.html'
+  license :bsd
+
+  app 'Pd-0.46-6-64bit.app'
+  
+  uninstall_preflight do
```
Comment (vitorgalvao) [inline Casks/pd.rb]: Why not simply `postflight`?

**Comment by megrimm** [comment]
hello... i made changes and pushed. should i create new pull request? this is my first caskroom contribution and I'm pretty new to git and currently learning the ropes.... 

Thanks!

**Comment by vitorgalvao** [comment]
I should’ve been more specific, but was on a phone, at the time. I didn’t mean replacing `uninstall_preflight` with `uninstall_postflight`, but with `postflight` entirely. Why make the changes before uninstalling, and not right after installing?

**Comment by megrimm** [comment]
got it... i had modeled this cask off the pd-extended cask which also might be wrong?

**Comment by vitorgalvao** [comment]
To be clear, I was not claiming it was wrong, but rather asking really why using one solution and not the other. Lets merge it this way and see if an issue occurs.

Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by vitorgalvao** [comment]
Done in #10358. Your contribution is still included and still credited to you. Thank you.

**Pull request closed without merging**